File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        List<Integer> list = new ArrayList<>();

        for (int i = 0; i < n; i++) {
             int num = sc.nextInt();
             list.add(num);
        }
        int v = sc.nextInt();
        int count = (int) list.stream().filter(a -> a == v).count();
        System.out.println(count);
    }
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.c
#include<stdio.h>
int main(void){
    int N;
    int a[100];
    int v;
    int count = 0;
    scanf("%d",&N);
    
    for(int i = 0; i < N; i++){
        scanf("%d",&a[i]);
    }
    scanf("%d",&v);
    
    for(int i = 0; i < N; i++){
        if(a[i] == v)
            count++;
    }
    
    printf("%d",count);
}